packages/astro/src/core/dev/index.ts

```diff
@@ -73,7 +73,7 @@ export class AstroDevServer {
     this.port = config.devOptions.port;
     this.origin = `http://localhost:${this.port}`;
     this.site = config.buildOptions.site ? new URL(config.buildOptions.site) : undefined;
-    this.pathname = this.site ? this.site.pathname + '/' : '/';
+    this.pathname = this.site ? this.site.pathname : '/';
```

Comment:
Tiny suggestion (not blocking the PR): what about `devRoot` or something that indicates it’s the base directory of the dev server? `this.pathname` may be confused for request pathname inside certain contexts (e.g. `this.pathname` alongside `pathname`)

Reply:
Good call, updated.